### PR TITLE
Create SI_V25_02 model (with IDs modelled as kickmaps)

### PR DESCRIPTION
**Pull request by xresende**
- Add insertion devices as kickmaps to pymodels. 
- Update pyaccel model to _SI_V25_02_

Example:

```python
#!/usr/bin/env python-sirius

import matplotlib.pyplot as plt
import pyaccel
from pymodels import si
from pymodels.SI_V25_02.idmodel import IDModel

FOLDER_BASE = '/home/ximenes/repos-dev/'
FOLDER_SIMUL = 'id-sabia/model-03/simulation/magnetic/'
FOLDER_SABIA = (FOLDER_BASE + FOLDER_SIMUL + 'fieldmaps_and_kickmaps_with_errors/')
POLARIZATION = 'horizontal_polarization/'
KVALUE = 'kmax/'

config = (
    '2021-08-14_DeltaSabia_RandomModel01_HorizontalPolarization' +
    '_Kh=6.1_Kv=0.0_X=-7_7mm_Y=-4_4mm_Z=-800_800mm.kck')

fname = FOLDER_SABIA + POLARIZATION + KVALUE + config
delta52 = IDModel(
        subsec=IDModel.SUBSECTIONS.ID10SB, file_name=fname,
        fam_name='DELTA52', 
        nr_steps=40, rescale_kicks=1.0, rescale_length=1.2/1.6)


ids = [delta52, ]

ring_bare = si.create_accelerator(ids=[])
ring_ids = si.create_accelerator(ids=ids)
print('bare ring length   : ', ring_bare.length)
print('with ID ring length: ', ring_ids.length)

tw0, _ = pyaccel.optics.twiss.calc_twiss(ring_bare)
tw1, _ = pyaccel.optics.twiss.calc_twiss(ring_ids)
dtw = 50*(tw1.betax - tw0.betax)
plt.plot(tw0.spos, tw0.betax, label='bare')
plt.plot(tw1.spos, tw1.betax, label='with ID')
plt.plot(tw0.spos, dtw, label='diff x 50')
plt.legend()
plt.xlabel('posx [m]')
plt.ylabel('BetaX [m]')
plt.show()
```
Figure_1

### CODE REVIEW

File: pymodels/SI_V25_02/lattice.py [hunk 1]
```diff
@@ -0,0 +1,865 @@
...
+    if ids:
+        idsdict = {id.subsec:id for id in ids}
+    else:
+        idsdict = dict()
+
+    kickmaps = dict()
+    
+    # CARNAUBA
+    idkey = 'ID06SB'
+    if ids is None or idkey not in idsdict:
```
Comment (fernandohds564) [inline pymodels/SI_V25_02/lattice.py:818]: @xresende, you can remove the condition `ids is None` from all if statements.

File: pymodels/__init__.py [hunk 2]
```diff
@@ -8,16 +8,17 @@
 from . import TS_V04_01
 from . import SI_V24_04
```
Comment (fernandohds564) [inline pymodels/__init__.py:9]: I think we can delete the folder `SI_V24_04` by now, can't we?

**Review by fernandohds564 — CHANGES_REQUESTED** [review]

File: pymodels/SI_V25_02/lattice.py (hunk 1 above)
Comment (xresende) [reply pymodels/SI_V25_02/lattice.py:818]: @fernandohds564, this function structure changed from the time you opened your review and when you committed. But the new structure has the same redundancy. I'll clean it.

File: pymodels/__init__.py (hunk 2 above)
Comment (xresende) [reply pymodels/__init__.py:9]: actually, I think we don't need to keep old accelerator versions, as we did in matlab. they are in code version control, anyway, and it is easy to restore them, if needed. what do you think, @fernandohds564 , @murilobalves ?

File: pymodels/SI_V25_02/lattice.py [hunk 3]
```diff
@@ -0,0 +1,865 @@
...
+    SS_S09 = IDA_09  # MANACA
+    SS_S10 = IDB_10  # SABIA
+    SS_S11 = IDP_11  # IPE
+    SS_S12 = IDB_12  # COLIBRI
+    SS_S13 = IDA_BbBKckrH
+    SS_S14 = IDB  # HARPIA
+    SS_S15 = IDP  # SAGUI
+    SS_S16 = IDB_16  # INGA
+    SS_S17 = IDA_17  # SAPUCAIA
+    SS_S18 = IDB_TunePkup  # PAINEIRA
```
Comment (murilobalves) [inline pymodels/SI_V25_02/lattice.py:420]: The beamline information regarding the corresponding straight section was recently updated [here](https://wiki-sirius.lnls.br/mediawiki/index.php/Machine:Insertion_Devices#Beamline_Insertion_Devices_Specifications) 

The difference from [this table](https://wiki-sirius.lnls.br/mediawiki/index.php/Machine:Storage_Ring#Straight_sections_allocation) is that PAINERA 18SB -> 14SB (replacing HARPIA)

COLIBRI, HARPIA, SAGUI and INGA beamlines are not mentioned in the new table.

**Review by murilobalves — CHANGES_REQUESTED** [review]

File: pymodels/__init__.py (hunk 2 above)
Comment (fernandohds564) [reply pymodels/__init__.py:9]: I agree @xresende !

File: pymodels/SI_V25_02/lattice.py (hunk 3 above)
Comment (xresende) [reply pymodels/SI_V25_02/lattice.py:420]: I updated this PR according to the wiki table. thanks!

File: pymodels/SI_V25_02/lattice.py (hunk 1 above)
Comment (fernandohds564) [reply pymodels/SI_V25_02/lattice.py]: humm, I didn't understand what you said, but if you'll clean the code its ok!

File: pymodels/__init__.py (hunk 2 above)
Comment (xresende) [reply pymodels/__init__.py:9]: ok. I'll add a commit.

**Review by fernandohds564 — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Review by jquentino — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Review by vellosok75 — APPROVED** [review]

**Pull request merged**